microblog/app/models.py

```diff
@@ -54,6 +55,12 @@ class User(UserMixin, db.Model):
     def __repr__(self):
         return '<User {}>'.format(self.username)
 
+    def avatar_url(self):
```

Comment:
The addition of the avatar_url method is a good enhancement for managing user avatars.